### PR TITLE
DEVEX-1387 python2 -> python3

### PR DESCRIPTION
**Pull request by emiloslavsky**
DEVEX-1387 python2 -> python3

### CODE REVIEW

File: Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ In order to take full advantage of the scalability that cloud computing offers,
 This applet tutorial code is extremely similar to the [_Parallel Threads SAMtools count tutorial_](/python_parallel_tutorial.html#samtools_count_para_chr_subprocess_py), except `multiprocessing` is used instead of `multiprocessing.dummy`.
 
 ## How is the SAMtools dependency provided?
-The SAMtools dependency is resolved by declaring an [Apt-Get](https://help.ubuntu.com/14.04/serverguide/apt-get.html) package in the `dxapp.json` `runSpec.execDepends` field.
+The SAMtools dependency is resolved by declaring an [Apt-Get](http://manpages.ubuntu.com/manpages/xenial/man8/apt-get.8.html) package in the `dxapp.json` `runSpec.execDepends` field.
 ```json
 {
   "runSpec": {
```
Comment (commandlinegirl) [inline Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md:15]: In L22:  "execDepends wiki page" --> "execDepends documentation"

File: Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ inputs
 ## Split workload
 We parallely process by using the python `multiprocessing` module. A rather simple pattern you can follow to compute in a parallel manner is shown below:
 ```python
-print "Number of cpus: {0}".format(cpu_count())  # Get cpu count from multiprocessing
+print("Number of cpus: {0}".format(cpu_count()))  # Get cpu count from multiprocessing
 worker_pool = Pool(processes=cpu_count())  # Create a pool of workers, 1 for each core
 results = worker_pool.map(run_cmd, view_cmds)  # map run_cmds to a collection
                                                # Pool.map will handle orchestrating the job
```
Comment (commandlinegirl) [inline Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md:45]: In L49: 

```
[python3 docs](https://docs.python.org/3/library/multiprocessing.html)
```

File: Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ In order to take full advantage of the scalability that cloud computing offers,
 This applet tutorial code is similar to the [_Parallel Cores SAMtools count tutorial_](/python_parallel_tutorial.html#samtools_count_para_reg_subprocess_py), except `multiprocessing.dummy` is used instead of `multiprocessing`.
```
Comment (commandlinegirl) [inline Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md:8]: This link is broken.

Comment (commandlinegirl) [inline Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md:8]: In this file the link "multiprocessing.dummy" should point to:

https://docs.python.org/3/library/multiprocessing.html#module-multiprocessing.dummy

File: Tutorials/python/samtools_count_distr_region_py/Readme.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-The applet will create a count of reads from a BAM format file. Documentation to create a distributed applet can be found on the [DNAnexus wiki](https://wiki.dnanexus.com/Developer-Tutorials/Parallelize-Your-App). This readme will focus on the details of this applet.
+The applet will create a count of reads from a BAM format file. Documentation to create a distributed applet can be found on the [DNAnexus documentation site](https://documentation.dnanexus.com/faqs/developing-apps-and-applets#how-do-i-parallelize-my-app). This readme will focus on the details of this applet.
```
[the remark this answers is not in the document]
Comment (emiloslavsky) [reply Tutorials/python/samtools_count_distr_region_py/Readme.md:1]: done

File: Tutorials/python/samtools_count_distr_region_py/Readme.md [hunk 5]
```diff
@@ -10,10 +10,10 @@ The SAMtools dependency is resolved by declaring an [Apt-Get](https://help.ubunt
     ]
   }
 ```
-For additional information, please refer to the [`execDepends` wiki page](https://wiki.dnanexus.com/Execution-Environment-Reference#Software-Packages).
+For additional information, please refer to the [`execDepends` wiki page](https://documentation.dnanexus.com/getting-started/tutorials/developer-tutorials/bash/git-dependency#how-is-the-samtools-dependency-added).
```
[the remark this answers is not in the document]
Comment (emiloslavsky) [reply Tutorials/python/samtools_count_distr_region_py/Readme.md]: done

File: Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ In order to take full advantage of the scalability that cloud computing offers,
 This applet tutorial code is extremely similar to the [_Parallel Threads SAMtools count tutorial_](/python_parallel_tutorial.html#samtools_count_para_chr_subprocess_py), except `multiprocessing` is used instead of `multiprocessing.dummy`.
```
[the remark this answers is not in the document]
Comment (emiloslavsky) [reply Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md]: done

File: Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md (hunk 1 above)
Comment (emiloslavsky) [reply Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md:15]: done

File: Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md (hunk 2 above)
Comment (emiloslavsky) [reply Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md:45]: done

File: Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md (hunk 3 above)
Comment (emiloslavsky) [reply Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md]: done

Comment (emiloslavsky) [reply Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md]: done

**Comment by emiloslavsky** [comment]
@commandlinegirl - if fixed up link issues in the 4 Readme I've modified.

File: Tutorials/bash/samtools_count_distr_chr_slice_sh/Readme.md [hunk 7]
```diff
@@ -1,20 +1,17 @@
-Documentation to create a distributed applet can be found on the [Developer Tutorials wiki page](https://wiki.dnanexus.com/Developer-Tutorials/Parallelize-Your-App). This readme will focus on the details of this applet.
+This readme describes the impelementation of a distributed applet.
```
Comment (commandlinegirl) [inline Tutorials/bash/samtools_count_distr_chr_slice_sh/Readme.md:1]:
```suggestion
This readme describes the implementation of a distributed applet.
```

**Pull request merged**